src/Domain/Creators/BinaryCompactObject.cpp

```diff
@@ -418,6 +432,10 @@ Domain<3> BinaryCompactObject::create_domain() const noexcept {
 
   // Inject the hard-coded time-dependence
   if (enable_time_dependence_) {
+    // Note on frames: Because the relevant maps will all be composed before
+    // they are used, all maps here go from Frame::Grid (the frame after the
+    // final time-independent map is applied) to Frame::Inertial
+    // (the frame after the final time-dependent map is applied).
```

Comment:
I thought we were using Frame::None for compositions.

Reply:
Are you saying I should switch Frame::Grid to Frame::None? If so, then what is Frame::Grid for?

Reply:
OK, it looks like you're right and the manipulation functions do require all the maps to be Grid->Inertial.  That doesn't make any sense mathematically, but I guess we go with it.

NoFrame (not None, I misremembered) is for intermediate unnamed frames, which seems like exactly what you have here.  (A frame after scaling but before rotation.)